src/browser/provider/index.ts

```diff
@@ -317,8 +319,11 @@ export default class BrowserProvider {
             else
                 await this._takeLocalBrowserScreenshot(browserId, screenshotPath);
```

Comment:
Looks like we won't create a directory in the code path where `_takeLocalBrowserScreenshot` is called.

Reply:
`testcafe-browser-tools` will
https://github.com/DevExpress/testcafe-browser-tools/blob/775f1012c454558a0fd58af1f576036e9b22c453/src/api/screenshot.js#L17

Reply:
It's very unobvious. At least, we should add a comment about it. 

But I would prefer to refactor the code to get both cases covered by `makeDir`.